ecl/eclcc/eclcc.cpp

```diff
@@ -483,12 +485,13 @@ int main(int argc, const char *argv[])
 
     unsigned exitCode = doMain(argc, argv);
 
-#ifndef _DEBUG
```

Comment:
Do we not want to preserve this behaviour for debug mode? I.e. make the default for optReleaseMem true for #ifdef _DEBUG.